bitmark-references/bitmark-node-software/mining.md

```diff
@@ -0,0 +1,123 @@
+# Overview
+
+This paragraph describes mining mechanism, including hashing, communication protocol, and rewards.
+
+# Block Diagram
+
+            +--------------+  zero mq protocol   +---------------+
+            |              |  ---------------->  |               |
+            |   bitmarkd   |                     |   recorderd   |
+            |              |  <----------------  |               |
+            +--------------+  zero mq protocol   +---------------+
+
```

Comment:
What's missing from this intro is any discussion of ming, the topic of this section. We need to bring that up before we dive into hashing.

Something like:
"The mining procedure for Bitmark occurs through interactions between `bitmarkd` and `recorderd`." (and then continue with the second sentence.)

Reply:
updated